docs/tutorials.rst

```diff
@@ -228,7 +232,278 @@ method to drop values we can't decode using ``utf-8``, or if the raw trytes
 can't be decoded into legit bytes. A possible reason for the latter can be if
 the attribute contains a signature rather than a message.
 
+4.a Generate Address
+--------------------
+
+In this example, you will learn how to:
+
+- **Generate a random seed.**
+- **Generate an IOTA address that belongs to your seed.**
+- **Acquire free devnet IOTA tokens that you can use to play around with.**
```

Comment:
❔ devnet or testnet?  I think in earlier tutorials we refer to it as testnet.

Reply:
The official docs call it [devnet](https://docs.iota.org/docs/getting-started/0.1/network/iota-networks), so I created this ticket #263. Planned to be in the next release.